be/src/storage/lake/local_pk_index_manager.cpp

```diff
@@ -86,8 +86,11 @@ void LocalPkIndexManager::gc(UpdateManager* update_manager, DataDir* data_dir, s
         // just remove if not.
         if (StorageEngine::instance()->get_persistent_index_store(id) != data_dir) {
             dir_changed_tablet_ids.push_back(id);
-            if (clear_persistent_index(id).ok()) {
+            auto st = clear_persistent_index(id);
+            if (st.ok()) {
                 removed_dir_tablet_ids.push_back(id);
+            } else {
+                LOG(ERROR) << "clear persistent index fail, st: " << st.to_string() << " tablet id: " << id;
```

Comment:
if it is not an RPC failure or a fail causes user query/function failure, the warning level would be prefered.

Also is it possible to consider VLOG and change the vlog level in runtime to help troubleshoot?

Reply:
You are right, I will change the way to do this.